internal/venice-client-common/src/test/java/com/linkedin/venice/schema/TestAvroSupersetSchemaUtils.java

```diff
@@ -411,6 +411,30 @@ public void testIsSupersetSchemaNestedRecordSchema() {
     Assert.assertFalse(AvroSupersetSchemaUtils.isSupersetSchema(schema1, schema2));
   }
 
+  @Test
+  public void testSupersetSchemaContainsMergeFieldProps() {
+    String valueSchemaStr1 = "{\n" + "  \"name\": \"TestRecord\",\n" + "  \"type\": \"record\",\n" + "  \"fields\": [\n"
```

Comment:
nit: Not need to happen in this PR, but I might eventually want to make a new way to create schema programmatically to make it more readable in legacy test classes. Probably built some simply value schema builder on top of the FieldBuilder.

Reply:
I made one for VPJ schema, but not general enough. But anyway, can evolve later.